servus/serializable.h

```diff
@@ -138,9 +143,8 @@ class Serializable
         { throw std::runtime_error( "JSON serialization not implemented" ); }
     //@}
 
-private:
-    ChangeFunc _updated;
-    ChangeFunc _requested;
+    class Impl;
+    Impl* _impl;
```

Comment:
You need to deactivate or implement the assignment operator and copy constructor, otherwise will lead to double free/missing free of _impl.